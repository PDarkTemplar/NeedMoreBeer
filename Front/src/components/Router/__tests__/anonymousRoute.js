// @flow
import { shallow } from 'enzyme';

import React from 'react';

import Route from '../AnonymousRoute';

test('should render route', () => {
    const route = shallow(<Route />);

    expect(route).toMatchSnapshot();
});
