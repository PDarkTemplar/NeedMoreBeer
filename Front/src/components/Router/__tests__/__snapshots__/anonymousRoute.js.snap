// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render route 1`] = `<Route />`;
